codecov.yml

```diff
@@ -1,8 +0,0 @@
-coverage:
-  status:
-    project:
-      default:
-        target: auto
-        # adjust accordingly based on how flaky your tests are
-        # this allows a 10% drop from the previous base commit coverage
-        threshold: 10%
```

Comment:
I think change coverage targets are valid for at least the simulation fixtures since we don't want those to necessarily drop. I agree that the jury is still out on the robot side since I'm not sure if unit-testing is even a worthwhile endeavor (e.g. unit-testing a potential map seems like more pain than it is actually worth).